vk_video_decoder/libs/NvVideoParser/src/VulkanH265Parser.cpp

```diff
@@ -2039,26 +2039,26 @@ void VulkanH265Decoder::hrd_parameters(hevc_video_hrd_param_s* pStdHrdParameters
 {
     if (commonInfPresentFlag)
     {
-        pStdHrdParameters->flags.nal_hrd_parameters_present_flag = u(1);
-        pStdHrdParameters->flags.vcl_hrd_parameters_present_flag = u(1);
+        pStdHrdParameters->flags.nal_hrd_parameters_present_flag = u(1) & 0x1;
```

Comment:
I'm not sure if u(1) would only set the last bit or would set any bit into the integer.
I would suggest everywhere you use
**u(1) & 0x1** 
replace it with:
**flag()** 
There is already a function for that I have created in the past.